parquet-hadoop/src/test/java/org/apache/parquet/hadoop/TestParquetWriter.java

```diff
@@ -142,6 +142,82 @@ public void test() throws Exception {
     }
   }
 
+  @Test
+  public void testQuotedColumns() throws Exception {
+    Configuration conf = new Configuration();
+    Path root = new Path("target/tests/TestParquetWriter/");
+    enforceEmptyDir(conf, root);
+    MessageType schema = parseMessageType(
+        "message test { "
+        + "required binary `binary field`; "
```

Comment:
Can you change some of these to test cases other than space? I'd like to see `.` tested, along with other common patterns.